src/packages/excalidraw/README.md

```diff
@@ -0,0 +1,216 @@
+### Excalidraw
+
+Excalidraw exported as a component to directly embed in your projects
+
+### Installation
+
+You can use npm
+
+```
+npm install react react-dom @excalidraw/excalidraw
+```
+
+or via yarn
+
+```cassandraql
```

Comment:
I think it's either `bash` or nothing in this case.. what's casandraql?

Reply:
This got introduced by autocomplete in my editor  :D (most probably it's for Apache Cassandra ), I have removed it.